src/watermarks.ml

```diff
@@ -28,7 +28,6 @@ let make_watermark_map ~name ~version ~commit =
     match Opam_file.get_field opam_file name with
     | None -> Error (sprintf "variable %S not found in opam file" name)
     | Some value ->
-      let err = Error (sprintf "invalid value for variable %S in opam file" name) in
```

Comment:
What about `let err () = ...`? That would avoid the duplication

Reply:
I can do that, but likely we should drop this PR and see if this is fixed upstream.

PR for trunk testing coming soon.

Reply:
Yep, waiting a little bit to see how it plays out upstream seems good